test/ldclient_eval_SUITE.erl

```diff
@@ -810,6 +812,101 @@ rule_match_rollout_not_in_experiment(_) ->
     ExpectedEvents = ActualEvents.
 
 all_flags_state(_) ->
```

Comment:
This test was supposed to be without reasons and the other test with reasons, but it originally had a mistake. So both tests were with reasons.